Demo/Sources/Cells/HostingConfigurationCell/HostingConfigurationCell.swift

```diff
@@ -3,9 +3,12 @@ import FinniversKit
 import SwiftUI
 
 final class HostingContentConfigurationCell: UITableViewCell {
+    var onUpdateCellsConfiguredCount: (() -> Void)?
```

Comment:
IMHO introducing this closure just for debug / demo purposes could lead to confusion (at least it did for me 😆). 

Wouldn't it be cleaner to make a specific subclass in the demo target? 🤔

Reply:
It is a subclass in the demo target. 😄 But the name should include demo in it, it's a bit confusing.

Reply:
Ohh man, I should get new glasses 👴 😆